mk/install-build-tools.sh

```diff
@@ -63,6 +63,12 @@ case $target in
 --target=i686-unknown-linux-musl|--target=x86_64-unknown-linux-musl)
   use_clang=1
   ;;
+--target=mipsel-unknown-linux-gnu)
+  install_packages \
+    qemu-user \
+    gcc-mipsel-linux-gnu \
+    libc6-dev-mipsel-cross
```

Comment:
How important is it for you to use GCC instead of clang?

Reply:
Not important at all, I didn't know that Cargo could link mipsel code without it 😄

Reply:
Cross image for mipsel uses the [following packages by default](https://github.com/rust-embedded/cross/blob/b7927f34a15ed21e13feae8899688428026a9f0a/docker/Dockerfile.mipsel-unknown-linux-gnu#L13-L14)